slices/iptables.yaml

```diff
@@ -0,0 +1,204 @@
+package: iptables
+
+essential:
+  - iptables_copyright
+
+slices:
+  bins:
+    essential:
+      - iptables_libs
+      - iptables_links
+      - libc6_libs
+      - libip4tc2_libs
+      - libip6tc2_libs
+      - libmnl0_libs
+      - libnetfilter-conntrack3_libs
+      - libnfnetlink0_libs
+      - libnftnl11_libs
+      - libxtables12_libs
+      - netbase_default-hosts
+      - netbase_default-networks
+    contents:
+      /usr/sbin/arptables-nft:
+      /usr/sbin/arptables-nft-restore:
+      /usr/sbin/arptables-nft-save:
+      /usr/sbin/ebtables-nft:
+      /usr/sbin/ebtables-nft-restore:
+      /usr/sbin/ebtables-nft-save:
+      /usr/sbin/ebtables-translate:
+      /usr/sbin/ip6tables-apply:
+      /usr/sbin/ip6tables-legacy:
+      /usr/sbin/ip6tables-legacy-restore:
+      /usr/sbin/ip6tables-legacy-save:
+      /usr/sbin/ip6tables-nft:
+      /usr/sbin/ip6tables-nft-restore:
+      /usr/sbin/ip6tables-nft-save:
+      /usr/sbin/ip6tables-restore-translate:
+      /usr/sbin/ip6tables-translate:
+      /usr/sbin/iptables-apply:
+      /usr/sbin/iptables-legacy:
```

Comment:
it is acceptable to do this later. but if it makes sense to your use case, it could also be done now. And that is:
 - is there a way we could split the libs and bins slices further? I.e., right now, when installing `bins`, you pretty much get the whole pkg. However, it seems like some `bins` could be nicely grouped by the type of functionality/domain they operate on. For example, we can have slices for `ip6tables`, `iptables`, `legacy`, etc. Would that make sense? (again, if you need them all today, then we can still create the splits later)

Reply:
For our use case today we need all of them, if we were to reduce this it would require us to understand what the consequences would be of not including this. Unfortunately we have customers that tend to use whatever they please so it's really hard for us to just drop these kinds of tools

Reply:
that's fair. if your use case requires all, then it's ok to start from a sub-optimal starting point. we can always slice it further whenever such a use case arises.